sdk/framework/backend_test.go

```diff
@@ -240,14 +240,17 @@ func TestBackendHandleRequest_badwrite(t *testing.T) {
 		},
 	}
 
-	_, err := b.HandleRequest(context.Background(), &logical.Request{
+	resp, err := b.HandleRequest(context.Background(), &logical.Request{
 		Operation: logical.UpdateOperation,
 		Path:      "foo/bar",
 		Data:      map[string]interface{}{"value": "3false3"},
 	})
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
 
-	if err == nil {
-		t.Fatalf("should have thrown a conversion error")
+	if resp.Data["error"] != "Field validation failed: error converting input 3false3 for field \"value\": cannot parse '' as bool: strconv.ParseBool: parsing \"3false3\": invalid syntax" {
```

Comment:
Maybe just look for contains "Field validation failed"?  Nice to not have to update tests just because you tweak the wording of an error message.

Reply:
Good call, thanks